src/coreclr/debug/ee/dactable.cpp

```diff
@@ -23,45 +23,133 @@
 
 #ifdef DEBUGGING_SUPPORTED
 
-extern PTR_ECHash gFCallMethods;
+extern PTR_ECHash gFCallMethods[FCALL_HASH_SIZE];
 extern TADDR gLowestFCall;
 extern TADDR gHighestFCall;
-extern PCODE g_FCDynamicallyAssignedImplementations;
+extern "C" PCODE g_FCDynamicallyAssignedImplementations;
 extern DWORD gThreadTLSIndex;
 extern DWORD gAppDomainTLSIndex;
+extern "C" void STDCALL ThePreStubPatchLabel(void);
 
-DLLEXPORT
-DacGlobals g_dacTable;
+namespace ABI
+{
+    struct ComInterfaceDispatch;
+}
+HRESULT STDMETHODCALLTYPE ManagedObjectWrapper_QueryInterface(
```

Comment:
Let's point to the real definition in interoplib by stating they are forward declarations. Do these need to be protected by `FEATURE_COMWRAPPERS`?

Reply:
Since they're forward declarations, they don't need to be protected with `FEATURE_COMWRAPPERS`, but that's probably a good idea anyway.